bigquery/google/cloud/bigquery/table.py

```diff
@@ -717,13 +717,21 @@ def _build_resource(self, filter_fields):
 
 
 class TableListItem(object):
-    """Read-only table resource object with a subset of table properties.
+    """A read-only table resource from a list operation.
+
+    For performance reasons, the BigQuery API only includes some of the table
+    properties when listing tables. Notably,
+    :attr:`google.cloud.bigquery.table.Table.schema` and
+    :attr:`google.cloud.bigquery.table.Table.num_rows` are missing.
+
+    For a full list of the properties that the BigQuery API returns, see the
+    `REST documentation for tables.list
+    <https://cloud.google.com/bigquery/docs/reference/rest/v2/tables/list>`.
```

Comment:
How does this multiline anchor render?

Reply:
Looks fine to me.

![image](https://user-images.githubusercontent.com/247555/33044440-d78fc474-cdfd-11e7-8ab6-aa358df81ce1.png)